tf2onnx/tfonnx.py

```diff
@@ -2255,7 +2303,7 @@ def rewrite_incomplete_type_support_rs5(g, ops):
 
 
 def rewrite_incomplete_type_support_rs6(g, ops):
-    return rewrite_incomplete_type_support(g, ops, ["ReduceSum", "Slice", "Split", "Tile", "Transpose"])
+    return rewrite_incomplete_type_support(g, ops, ["Div", "Greater", "ReduceSum", "Slice", "Split", "Tile", "Transpose"])
```

Comment:
We find it is a bug related to rewrite_incomplete_type_support, as current implementation assumes the type of inputs and outputs of nodes that need to be rewritten is always the same. 
@zhijxu-MS will try to fix it and then remove my previous Greater support since it should be covered by the amended rewriter.

Reply:
I mean the Greater PR submitted by me is actually a bug related to rewrite_incomplete_type_support.

Reply:
I guess this was by design, when this case was not considered. feel free to enhance it if you have time. and add author @guschmue  in the review.